contracts/Remittance.sol

```diff
@@ -34,11 +34,10 @@ contract Remittance is Stoppable {
         require(transactions[hashedOTP].emitter == address(0), "Password already used");
         
         uint amount = msg.value;
-        if (getOwner() != msg.sender) {
-            fees[getOwner()] = fees[getOwner()].add(TX_FEES);
-            amount = amount.sub(TX_FEES);
-            emit LogTakeFees(msg.sender, TX_FEES);
-        }
+
+        fees[getOwner()] = fees[getOwner()].add(TX_FEES);
+        amount = amount.sub(TX_FEES);
```

Comment:
I think you can save gas by replacing:

```solidity
uint amount = msg.value;
amount = amount.sub(TX_FEES);
```
with:

```solidity
uint amount = msg.value.sub(TX_FEES);
```

Reply:
Well spotted!